src/main/java/bio/terra/app/controller/JobsApiController.java

```diff
@@ -97,10 +100,13 @@ public ResponseEntity<List<JobModel>> enumerateJobs(
       Integer offset,
       Integer limit,
       @RequestParam(defaultValue = "desc") SqlSortDirection direction,
```

Comment:
Probably a good idea to remove `@RequestParam` while you're here
```suggestion
      SqlSortDirection direction,
```

Reply:
We can't do that sadly.  This is why we need to upgrade to OAS3.1.  We need to explicitly set it to descending since the code gen doesn't specify a default